src/iocore/cache/StripeSM.cc

```diff
@@ -164,19 +164,15 @@ StripeSM::init(char *s, off_t blocks, off_t dir_skip, bool clear)
            static_cast<uint64_t>(dir_skip), static_cast<uint64_t>(blocks));
   CryptoContext().hash_immediate(hash_id, hash_text, strlen(hash_text));
 
-  dir_skip = ROUND_TO_STORE_BLOCK((dir_skip < START_POS ? START_POS : dir_skip));
```

Comment:
Where did this go? Is there a reason it can be removed?

Reply:
It's here. This method doesn't use modified `dir_skip`, right?
https://github.com/apache/trafficserver/pull/11742/files#diff-88d9e07d3781ce4d9c6307114be7856582eba25edc0d1c64ac9a73a59d00baecR285

Reply:
I see now, the temporary assignment was removed and `skip` is calculated directly. Nice improvement.